ConwaysGameOfLife/GameController.cs

```diff
@@ -39,42 +39,42 @@ public GameController(IInput input, IOutput output)
             _output = output;
         }
 
+        public void Run()
+        {
+            _output.WriteLine(Messages.Welcome);
+            CreateInitialUniverse();
+            DisplayUniverse();
+            // add live cells to universe until user presses 'p' to play
+            // PopulateUniverseWithLiveCells();
+            // DisplayUniverse();
+            // generator checks all cells if change of state required & creates next universe
+            // loop last step until user presses 'q' to quit or all cells are dead
+        }
+
         public void CreateInitialUniverse()
         {
             _output.WriteLine(Messages.RequestDimensions);
             var input = CreateValidUniverseString();
             _universe = InputParser.ParseUniverse(input);
         }
 
-        public void PopulateUniverseWithLiveCells()
-        {
-            // _output.Write(Messages.RequestLiveCell);
-            // _output.WriteLine($ or {Messages.Play});
-            // loop
-            // var locationInput = _input.ReadLine();
-            // var location = SetLiveCellLocation(locationInput, _universe.GridWidth, _universe.GridLength);
-            // universe.SwitchCellState(universe.GetCellAtLocation(location.X, location.Y));
-            throw new NotImplementedException();
-        }
-
-        public void DisplayUniverse() 
+        public void DisplayUniverse()
         {
             _output.WriteLine(OutputFormatter.FormatUniverse(_universe));
         }
 
-        public void Run()
+        public void PopulateUniverseWithLiveCells()
         {
-            _output.WriteLine(Messages.Welcome);
-            // create initial universe
-            CreateInitialUniverse();
-            DisplayUniverse();
-            // add live cells to universe until user presses 'p' to play
-            // PopulateUniverseWithLiveCells();
-            // DisplayUniverse();
-            // generator checks all cells if change of state required & creates next universe
-            // loop last step until user presses 'q' to quit or all cells are dead
+            _output.Write(Messages.RequestLiveCell);
+            _output.WriteLine($"or {Messages.Play}");
+            //loop until player presses "p" to play
+            var locationInput = CreateValidLocationString();
+            var location = InputParser.ParseLocation(locationInput);
+            _universe.SwitchCellState(_universe.GetCellAtLocation(location.X, location.Y));

```

Comment:
can we make the `GetCellAtLocation` function take a `Location` as a param?
```suggestion
            _universe.SwitchCellState(_universe.GetCellAtLocation(location.X, location.Y));

```